src/it/mjmod-8-generate-jmod-in-other-project/about-cli-distribution-jmod/pom.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements. See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership. The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License. You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied. See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jmod-plugin-mjmod-8</artifactId>
+        <version>99.0</version>
+    </parent>
+
+    <artifactId>about-cli-distribution-jmod</artifactId>
+    <packaging>jmod</packaging>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>about-cli-app</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jmod-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <classPath>${project.parent.basedir}/about-cli-app/target/about-cli-app-${project.version}.jar</classPath>

```

Comment:
There should be no need for this classpath entry. `about-cli-app` is a dependency, so will be available as a pathElement.

Reply:
I can replace this *classPath* entry with *project.dependencyArtifacts*. In the test case provided, pathElement is null because this Maven project just get a modular JAR and create a JMOD file from it. What ssems reasonable to do in this case is to fill *classPath* with *project.dependencyArtifacts*.